src/Neo/Cryptography/Hasher.cs

```diff
@@ -9,24 +9,21 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
-namespace Neo.SmartContract.Native
+namespace Neo.Cryptography
 {
     /// <summary>
-    /// Represents the named curve used in ECDSA.
+    /// Represents hash function identifiers supported by ECDSA message signature and verification.
     /// </summary>
-    /// <remarks>
-    /// https://tools.ietf.org/html/rfc4492#section-5.1.1
-    /// </remarks>
-    public enum NamedCurve : byte
+    public enum Hasher : byte
```

Comment:
Maybe rename to `NamedHashAlgorithm`

Reply:
This enum is not related to the `NamedCurveHash` and native contract, and I'd like it not to associate with `NamedCurveHash`. Thus, `Hasher` is kept.